index.d.ts

```diff
@@ -1,14 +1,14 @@
 declare module 'react-native-steve' {
-	import React from 'react'
-	import {ViewStyle} from 'react-native'
+	import { ViewStyle } from 'react-native';
 	
-	export type SteveProps = {
-		data: any[],
-		renderItem: ({item: any, index: number}) => React.ReactNode,
-		containerStyle?: ViewStyle
+	export interface SteveProps<T> {
+		data: T[],
+		renderItem: ({item, index}: {item: T, index: number}) => JSX.Element,
+		keyExtractor: (item: T) => string,
```

Comment:
This prop is not necessary anymore. I use index value of item to use it as a key.

Reply:
You'll likely not want to use the index value as the key for the same reason that FlatList also requires an explicit key. It's up to you, but I'd suggest following convention.

Reply:
Yea, you're right. Using index is an anti-pattern. I forget it